lib/kvdb/sched_sts.c

```diff
@@ -43,6 +43,12 @@ sts_job_run(struct work_struct *work)
     job->sj_job_fn(job);
 }
 
+int
+sts_jobcnt(struct sts *self)
```

Comment:
Take a const.

Reply:
Yeah, just whack this new API and use the sp start/finished counters...

Reply:
I removed this API and will use GB's idea to rely on `started==finished`.